svg/intro/script-content.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test: </title>
+<link rel="help" href="https://svgwg.org/svg2-draft/intro.html#AboutSVG">
+<meta name="assert" content="The same scripts can work on both HTML and SVG elements.">
+<meta name="assert" content="The script element is in the content model of all elements.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<script>var total = 0;</script>
+<script>function s0() { total += 100; return s1(); }</script>
+<svg>
+  <script>function s1() { total += 20; return s2(); }</script>
+  <g>
+    <script>function s2() { return s3(); }</script>
+    <circle id="spot">
+      <script>function s3() { return s4(); }</script>
+    </circle>
+  </g>
+  <desc>
+    <script>function s4() { return s5(); }</script>
+  </desc>
+  <filter>
+    <script>function s5() { return s6(); }</script>
+    <feFlood>
+      <script>function s6() { return s7(); }</script>
+    </feFlood>
+  </filter>
+  <text>
+    <script>function s7() { return s8(); }</script>
+    <tspan>
+      <script>function s8() { return s9(); }</script>
+    </tspan>
+  </text>
+  <use href="spot">
+    <script>function s9() { return s10(); }</script>
+  </use>
+</svg>
+<div id="box">
+  <script>function s10() { total += 3; return s11(); }</script>
+  <span>
+    <script>function s11() { return 'same'; }</script>
+  </span>
+</div>
+
+<script>
+test(function(){
+  assert_equals(s0(), 'same');
+  assert_equals(total, 123);
+
+  for (let element of [spot, box]) {
+    element.style.opacity = '0.75';
```

Comment:
I'm a little worried that this bit of code could get spurious fails in non-browser environments, from

- expecting elements with `id` values to be available in the global scope (not really sure if this is defined anywhere that SVG normatively references)
- expecting the `style` property to be accessible as a dictionary. This _is_ normative in the versions of CSSOM that SVG 2 references, but it wasn't in the specs referenced by SVG 1.1. So, for example, it would fail in Batik last I checked. And that's really a separate test/failure from the main purpose of this test (scripts are working and the entire HTML+SVG document shares a global scope).

Reply:
I can use `getElementById`.

With regards to property, the purpose of the code is to check that the same scripts can operate on SVG and CSS elements. I can use `lastChild` / `removeChild()` / `firstChild` / `insertBefore()` to achieve this.

Reply:
Yeah, something simple that is defined in core DOM. Could just be set & get an attribute.

Reply:
Since the root element is html and given the purpose of the test, not working in non-browser implementations is fine here.

A similar test with a chain of script elements as children of assorted svg-namespace elements, and with an svg root, would also be a useful test. (Not a blocker on this one, obviously).